lib/resources/utils.dart

```diff
@@ -55,9 +56,16 @@ class Utils {
 
   static Future<void> signOut(BuildContext context) async {
     final authProvider = Provider.of<AuthProvider>(context, listen: false);
-    unawaited(AppNavigator.pushNamedAndRemoveUntil(
-        context, Routes.login, (route) => false));
-    unawaited(authProvider.signOut());
+    // TODO(IoanaAlexandru): Might actually be better to use the same login on mobile?
```

Comment:
IIRC I did it like this because some views would break when we no longer had a user (cue the "Something went wrong"s). Is this not still the case? And is it not the case for web?

Reply:
the issue here was that we use `authProvider.isAuthenticated` as a guard not to navigate to `LoginPage` while authenticated, which translates into using or not the `WebShell`. thus, when signing out, until the process is completed, the `LoginPage` will load into the `WebShell`, it's really ugly, believe me.